src/result.jl

```diff
@@ -528,3 +547,96 @@ function LinearSystemColoringResult(
         T_factorization,
     )
 end
+
+## Bicoloring result
+
+function remap_colors(color::Vector{Int})
```

Comment:
I understand what the goal of this function but we should add more comments for the code.